stable/redis-ha/templates/redis-master-appliance-vpc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}-master-appliance-vpc
+  annotations:
+    cloud.google.com/load-balancer-type: "Internal"
```

Comment:
Can we factor this out to the values file?

Here is a good example:
https://github.com/kubernetes/charts/blob/master/stable/prometheus/values.yaml#L139
https://github.com/kubernetes/charts/blob/master/stable/prometheus/templates/server-service.yaml#L6

Reply:
Done.